database/migrations/2017_07_30_205523_create_reciters_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRecitersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reciters', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->text('description');
+            $table->integer('hits');
+            $table->string('image_path');
+            $table->smallInteger('status')->default(0);
```

Comment:
@szainmehdi the status fields is used by the package "hootlex/laravel-moderation"
0 is pending
1 is approved
2 is rejected
3 is postponed